experimental/StandardFiniteFields/test/runtests.jl

```diff
@@ -0,0 +1,44 @@
+using Test
+using Oscar
```

Comment:
```suggestion
```
You do not need these two lines, as the test procedure or `Oscar.test_experimental_module` handle that.